JS/Lesson34/goksel/recipe-app/src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { Link } from "react-router-dom";
+import Box from '@mui/material/Box';
+import { AppBar, Button, Stack, Toolbar, Typography } from "@mui/material";
+import HomeIcon from '@mui/icons-material/Home';
+import PersonIcon from '@mui/icons-material/Person';
+import SettingsIcon from '@mui/icons-material/Settings';
+import CreateIcon from '@mui/icons-material/Create';
+import LogoutIcon from '@mui/icons-material/Logout';
+
+interface NavbarProps {
+    isSignedIn: boolean;
+    setIsSignedIn: any
+}
+
+export const Navbar = ({ isSignedIn, setIsSignedIn }:NavbarProps) => {
+
```

Comment:
Don't forget to format files, there are too many empty lines left

Reply:
Fixed